File: 백준/Silver/14425. 문자열 집합/문자열 집합.py
import sys
input = sys.stdin.readline

n,m=map(int,input().split())

s = [input().rstrip() for i in range(n)]

s2 = [input().rstrip() for i in range(m)]
cnt=0
for i in s2:
    if i in s:
        cnt+=1
print(cnt)

File: 백준/Silver/14425. 문자열 집합/문자열 집합.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

let [N, M] = input[0].split(' ').map(Number);
let arrS = new Set(input.slice(1, N+1));
let arrM = input.slice(N+1);
let answer = 0;
for(let i=0; i<M; i++){
    if(arrS.has(arrM[i])) answer++
}
console.log(answer)